frontend/src/App.js

```diff
@@ -15,6 +15,7 @@ import * as project from 'core/project';
 import * as resource from 'core/resource';
 import * as stats from 'core/stats';
 import { UserControls } from 'core/user';
+import { BatchEditor } from 'modules/batcheditor';
```

Comment:
I think it will be confusing to call this an "editor", as it will overlap with the `genericeditor` and `fluenteditor` modules. If I were to read the list of modules I would assume `batcheditor` was the same kind of module as the other two, which it isn't. Maybe change it to `batchactions`?